src/app/modules/form/form.utils.ts

```diff
@@ -149,3 +149,10 @@ export const extractFormLinkView = <T extends IFormDocument>(
     link: `${appUrl}/${_id}`,
   }
 }
+
+/**
+ * Regex to to detect invalid-encoded utf-8 characters in stringified form field input
+ * Matches any sequence which starts with a non-backslash, an odd number of backslashes, followed by unicode escape sequence
+ * See https://mathiasbynens.be/notes/javascript-escapes for regex on unicode escape sequences
+ */
+export const UNICODE_ESCAPED_REGEX = /[^\\](\\\\)*\\u[0-9a-fA-F]{4}/
```

Comment:
/justANote: I was about to comment that this regex won't capture sequences at the beginning of inputs (since it consumes at least one leading character), but of course when used exclusively on stringified input, a first character (the double quote `"`) is guaranteed to exists, so it should be fine.

Reply:
thanks! it's exactly what I had in mind as well :)